components/pagination/collections/paging_collection.js

```diff
@@ -377,6 +373,9 @@
                 }
             }
         });
+
+        var PagingCollectionProto = PagingCollection.prototype;
```

Comment:
It's more the 2 ```var``` declarations I mean. If you add a comma to line 375 instead of a ```;``` then you could remove the ```var``` from this line.